WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerFragment.kt

```diff
@@ -260,11 +260,16 @@ class MediaPickerFragment : Fragment() {
                         requestStoragePermission()
                     }
                 }
+
+                recycler.setEmptyView(null)
+                actionable_empty_view.visibility = View.GONE
```

Comment:
I think the correct solution would be to move this logic to the `PhotoListViewModel`. We can introduce a new subclass (we have `Data`, `Empty` and we can add `Hidden`). The `build` method can decide to set it to `Hidden` based on the same check used `if (softAskRequest != null && softAskRequest.show)`. This moves the decision out of the fragment. WDYT?

Reply:
I think you are right 👍 !